### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by melashu**
# In this milestone I have made the following changes 

- Create all console app entry-point methods inside `app.rb` file
- Created entry point method inside `main. rb`

### CODE REVIEW

File: lib/main.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  app = App.new
+
+  choices = 1
+  until choices.eql? 7
+    puts 'Please choose an optional by enter in a number:'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
```
Comment (muneebulrehman) [inline lib/main.rb:14]: While trying to create a rental for any number other than 0, your application runs into an infinite loop. Kindly fix the issue. You can view [this](https://www.loom.com/share/776d33a9008d4434a71944fa5efcc665) video for further clarification, where i have tried to explain the problem in detail.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @melashu , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: lib/main.rb (hunk 1 above)
Comment (melashu) [reply lib/main.rb:14]: Hello @muneebulrehman, Thank you for your feedback. I think I already implement that. When you try to enter a number outside the range, it asks you to enter again. 
image

**Review by Ismailco — APPROVED** [review]
### Hi @melashu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**